### PR TITLE
meeting notes for the db meeting

### PR DESCRIPTION
**Pull request by aidnurs**
## Basics

- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

File: doc/meetings/2023_04_07_db.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    -   ?
+-   who wants to work on which task?
+
+## Notes
+
+-   Thorben presented the map
+-   re-check the requirements
+-   coordinates in the database
+-   can use different coordinate systems. but costly
+-   kepler.gl can use lang/lat coordinate systems
```
Comment (badnames) [inline doc/meetings/2023_04_07_db.md:85]:
```suggestion
-   kepler.gl can use long/lat coordinate systems
```

File: doc/meetings/2023_04_07_db.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+-   LeafletJS
+-   KonvaJS
+-   cartesian vs lang/lat coordinates
```
Comment (badnames) [inline doc/meetings/2023_04_07_db.md:88]:
```suggestion
-   cartesian vs long/lat coordinates
```

Comes from longitude/latitude

**Review by badnames — APPROVED** [review]
Just two minor spelling errors.

**Review by Benjaminimal — APPROVED** [review]

**Review by markus2330 — CHANGES_REQUESTED** [review]
I think everything is already contained in #227, so we can close this?

**Comment by aidnurs** [comment]
Closing since the meeting notes are already contained in #227

**Pull request closed without merging**